doc/manual/packages/binary-cache-substituter.xml

```diff
@@ -57,13 +57,15 @@ of Firefox, Nix will first check if the path is available on the
 server <literal>avalon</literal> or another binary caches. If not, it
 will fall back to building from source.</para>
 
-<para>You can also tell Nix to always use your binary cache by adding
+<para>You can also tell Nix to always use your binary cache, if using Nixos by setting the `nix.extraOptions` in `configuration`, otherwise by adding
```

Comment:
```suggestion
<para>You can also tell Nix to always use your binary cache. When using Nixos by setting the `nix.extraOptions` in `configuration`, otherwise by adding
```

Reply:
Well, changing it to ". When" creates a sentence without a main clause. But the ", if" isn't great stylistically either.

Also "NixOS" rather than "Nixos".